backend/src/zimfarm_backend/main.py

```diff
@@ -1,91 +1,110 @@
-import logging
-import os
-import pathlib
-
-# import boto3
-from flask import Flask, Response, make_response, redirect, render_template
-from flask_cors import CORS
-
-from routes import (
-    API_PATH,
-    auth,
-    errors,
-    languages,
-    offliners,
-    platforms,
-    requested_tasks,
-    schedules,
-    status,
-    tags,
-    tasks,
-    users,
-    workers,
-)
-from utils.database import Initializer
-from utils.json import ZimfarmJsonProvider
-
-# TEMP fix awaiting kiwixstorage
-# boto3.set_stream_logger("", logging.CRITICAL)
-
-if os.getenv("DOCS_DIR"):
-    docs_dir = pathlib.Path(os.getenv("DOCS_DIR")).resolve()
-else:
-    # docs dir outside codebase
-    docs_dir = pathlib.Path(__file__).parent.resolve().parent.joinpath("docs")
-application = Flask(__name__, template_folder=docs_dir)
-application.json = ZimfarmJsonProvider(application)
-CORS(application)
-
-logging.basicConfig(
-    level=logging.DEBUG, format="[%(asctime)s: %(levelname)s] %(message)s"
-)
-logger = logging.getLogger(__name__)
-
-
-@application.route(f"{API_PATH}/openapi.yaml")
-def openapi():
-    fname = "openapi_v1.yaml"
-    if not docs_dir.joinpath(fname).exists():
-        return Response("Zimfarm API (OpenAPI spec missing)")
-    resp = make_response(render_template(fname))
-    resp.headers["Content-Type"] = "application/yaml"
-    return resp
-
-
-@application.route(f"{API_PATH}")
-def api_doc():
-    fname = "swagger-ui.html"
-    if not docs_dir.joinpath(fname).exists():
-        return Response("Zimfarm API (documentation missing)")
-    return render_template(fname)
-
-
-@application.route("/")
-def home():
-    return redirect("/v1")
-
+# import logging
```

Comment:
What does mean all this commented code? Do I get it correctly that you hard to remove it for now but intend to put it back in place a bit later in the migration process?